requirements.adoc

```diff
@@ -272,20 +357,20 @@ ACCEPTANCE CRITERIA::
 
 * Verify that a referring physician can enter a new radiology order by entering
 ** patient
-** an orderable from concepts (i.e. "R/O pulmonary embolism")
+** an orderable from concepts eg CT head with contrast ~~(i.e. "R/O pulmonary embolism")~~
 ** urgency [choose from EMERGENCY, ROUTINE, ON SCHEDULED DATE;
 defaults to ROUTINE]
 ** date and time
 ** orderer from providers (in case I am not an RP, but a data clerk with
 RP permissions entering the order for an RP)
 ** order reason as free text
-** order reason from concept
+** order reason from concept eg ICD 9 diagnosis code
```

Comment:
ok, so I will just add the fields orderReasonCoded and orderReasonNonCoded to the order entry page